eng/scripts/Pack-Crates.ps1

```diff
@@ -156,64 +114,73 @@ function Create-ApiViewFile($package) {
   "$packagePath/review/$packageName.rust.json"
 }
 
-Push-Location $RepoRoot
+$originalLocation = Get-Location
 try {
-  $localRegistryPath = Initialize-VendorDirectory
+  Set-Location $RepoRoot
```

Comment:
1. Just use `push-location`. Why do it manually?
2. Why change directories at all? We should be using `cargo <verb> --package <package> ...` everywhere. Almost all `cargo` commands take a package argument and location doesn't matter.